spiceaidocs/docs/data-connectors/delta-lake.md

```diff
@@ -0,0 +1,138 @@
+---
+title: 'Delta Lake Data Connector'
+sidebar_label: 'Delta Lake Data Connector'
+description: 'Delta Lake Data Connector Documentation'
+pagination_prev: null
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Query/accelerate [Delta Lake](https://delta.io/) tables in Spice.
+
+## Configuration
+
+`spice login delta_lake` can be used to configure the secrets needed for connecting to Delta Lake tables.
+
+<Tabs>
+  <TabItem value="delta_lake_s3" label="Delta Lake + S3">
+    ```bash
+    spice login delta_lake --aws-region <aws-region> --aws-access-key-id <aws-access-key-id> --aws-secret-access-key <aws-secret-access-key>
+    ```
+
+  </TabItem>
+  <TabItem value="delta_lake_azure" label="Delta Lake + Azure Blob">
+    ```bash
+    spice login delta_lake --azure-storage-account-name <account-name> --azure-storage-access-key <access-key>
+    ```
+
+  </TabItem>
+  <TabItem value="delta_lake_gcp" label="Delta Lake + Google Storage">
+    ```bash
+    spice login delta_lake --google-service-account-path /path/to/service-account.json
+    ```
+
+  </TabItem>
+</Tabs>
+
+## Example
+
+```yaml
+datasets:
+  # Example for local Delta Lake
+  - from: delta_lake:/path/to/local/delta/table  // A local filesystem path to a Delta Lake table
```

Comment:
`delta_lake` connector should be added to index page as well: https://github.com/spiceai/docs/blob/trunk/spiceaidocs/docs/data-connectors/index.md